### PR TITLE
Changed initrd and added grub config definitions.

### PR DESCRIPTION
**Pull request by joachimmetz**
[Review on Reviewable](https://reviewable.io/reviews/forensicartifacts/artifacts/158)

### CODE REVIEW

File: definitions/linux.yaml [hunk 1]
```diff
@@ -186,12 +186,26 @@ labels: [System, Configuration Files]
 supported_os: [Linux]
 urls: ['http://en.wikipedia.org/wiki/Fstab']
 ---
+name: LinuxGrubConfiguration
+doc: Linux grub configuratoin file.
```
Comment (destijl) [inline definitions/linux.yaml]: "configuration"

**Comment by destijl** [comment]
LGTM, just a typo.

File: definitions/linux.yaml (hunk 1 above)
Comment (joachimmetz) [inline definitions/linux.yaml]: changed

**Comment by destijl** [comment]
I think we need to start squashing commits to keep the history clean, do you mind doing that?

**Comment by joachimmetz** [comment]
> I think we need to start squashing commits to keep the history clean, do you mind doing that?

Know that you can do this yourself as well for the merge:
```
git pull --squash url branch
git commit -a -m message --author name + email 
```

If you want to go that route, although I think it is too late for that, I opt to tweak plaso's review.py scripts to our needs.

**Pull request merged**